docs/lib/auth/fragments/ios/signin_web_ui/40_private_session.md

```diff
@@ -0,0 +1,9 @@
+### Prefer private session during signIn
+
+Starting Amplify 1.6.0, `Amplify.Auth.signInWithWebUI` automatically uses `ASWebAuthenticationSession` internally for iOS 13.0+. For older iOS versions, it will fall back to `SFAuthenticationSession`.
```

Comment:
any chance we have an Apple link to add here for customers who want to go deeper?

Reply:
Added